test/cucumber/InformarPontos.feature

```diff
@@ -0,0 +1,25 @@
+Feature:Informar pontos a coletar
+    As um coletor de res�duos
+    So that eu possa saber onde ir fazer a coleta
+    I want to ver e imprimir uma lista de pontos a coletar
```

Comment:
está com posição trocada com a linha 3, o certo seria "I want to...", "So that..."